component-lib/src/streamlit.ts

```diff
@@ -19,11 +19,31 @@
 import { EventTarget } from "event-target-shim";
 import { ArrowDataframeProto, ArrowTable } from "./ArrowTable";
 
+/** Configurable theme colors. */
+interface ThemeColors {
+  primary: string;
+  secondary: string;
+  bgColor: string;
+  secondaryBg: string;
+  bodyText: string;
+}
+
+/** Configurable theme fonts. */
+interface ThemeFonts {
+  bodyFont: string;
+}
+
+/** Object defining the currently set theme. */
+export interface Theme {
+  colors: ThemeColors;
+  genericFonts: ThemeFonts;
+}
```

Comment:
I wonder if we should make a separate type file just to separate things out? No strong preference at this stage but if component lib grows larger, we should probably do so then?

Reply:
Will keep this in mind in case `component-lib` continues to grow. For now, I'll leave things as-is in the interest of speed / since this seems like the first time `component-lib` has been edited since September so it may be awhile before these files get edited a lot again.